### DISCUSSION TITLE
Wayland applications do not respect GTK3 theme

### DISCUSSION THREAD

**Discussion by ghost** [hyprwm/Hyprland#339] [Q&A] [answered]
On the desktop, when hovering over the background, my cursor theme works as expected. However, when I hover over Waybar, Firefox, or PCManFM, the GTK3, cursor, and icon themes all revert to Adwaita. Is this a bug with my system, Hyprland, or GTK3? If it's a bug with Hyprland, I'll open a relevant issue.

**Comment by vaxerski** [comment, marked as answer]
https://github.com/hyprwm/Hyprland/wiki/FAQ#how-do-i-change-me-mouse-cursor

if that is not set properly gtk will revert to adwaita

**Reply by ghost** [reply]
My mouse cursor is set by LXAppearance to be Breeze. I manually checked my file in ~/.confiig/gtk-3.0/settings.ini, and everything is set correctly. Am I missing anything?

**Reply by vaxerski** [reply]
![image](https://user-images.githubusercontent.com/43317083/178123150-b0449b55-92f8-4932-8e20-327b901c82b3.png)
check these.

**Reply by ghost** [reply]
All are set correctly.

**Reply by vaxerski** [reply]
has to be an issue with your system then, as it works correctly for everyone else. Try restarting your pc after those changes. I remember I had issues with this too, but ultimately setting all the system configs and restarting worked.

**Reply by ghost** [reply]
I can fix the issue if I use gsettings. My guess is that this page's explanation applies to Hyprland as well: [https://github.com/swaywm/sway/wiki/GTK-3-settings-on-Wayland](https://github.com/swaywm/sway/wiki/GTK-3-settings-on-Wayland).

**Reply by vaxerski** [reply]
possible. Will add to the wiki.

**Reply by vaxerski** [reply]
![image](https://user-images.githubusercontent.com/43317083/178124045-96a04324-2e2d-4a01-8cc7-d63d200eda3f.png)

lmao i mentioned it already

**Reply by ghost** [reply]
Whoops, my bad.
